### PR TITLE
Homework9: Valeria Kudelko

### PR DESCRIPTION
**Pull request by ValeriaKudelko**

### CODE REVIEW

File: fifthfile.py
Comment (Stml89) [inline fifthfile.py:1]: `fifthsfile` is not a good filename

Comment (Stml89) [reply fifthfile.py:1]: use separate file for each task

File: fifthfile.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        else:
+            result.append(i)
+    return "".join(result)
+
+
+print(delete_character("a#bc#d"))
+print(delete_character("abc#d##c"))
+print(delete_character("abc##d######"))
+print(delete_character("#######"))
+print(delete_character(""))
```
Comment (Stml89) [inline fifthfile.py]: use assertion here to check that code works as expected

File: fifthfile.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            if count > 1:
+                result_one.append(string_one[k - 1] + str(count))
+            else:
+                result_one.append(string_one[k - 1])
+            count = 1
+    return ''.join(result_one)
+
+
+# Example: "abcde" == "abcde"
+string_exam = "abcde"  # pylint: disable=C0103
```
Comment (Stml89) [inline fifthfile.py]: for `pylint: disable=C0103` please use a separate file `.pylintrc` file

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: fifthfile.py
Comment (ValeriaKudelko) [reply fifthfile.py:1]: deleted the fifthfile.py, created new three files for each task

File: fifthfile.py (hunk 1 above)
Comment (ValeriaKudelko) [reply fifthfile.py]: ready

File: fifthfile.py (hunk 2 above)
Comment (ValeriaKudelko) [reply fifthfile.py]: ready

File: homework9.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        if i == "#" and result:
+            result.pop()
+        else:
+            result.append(i)
+    return "".join(result)
+
+
+assert delete_character("a#bc#d")  # True
+assert delete_character("abc#d##c")  # True
+# assert delete_character("abc##d######")  # AssertionError
```
Comment (Stml89) [inline homework9.py]:
```suggestion
assert not delete_character("abc##d######")
```

File: homework9.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+assert delete_character("#######")  # True
+# assert delete_character("")  # AssertionError
```
Comment (Stml89) [inline homework9.py]:
```suggestion
assert not delete_character("")
```

File: homework9part3.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+            else:
+                result_one.append(string_one[k - 1])
+            count = 1
+    return ''.join(result_one)
+
+
+# Example: "abcde" == "abcde"
+string_exam = "abcde"
+conclusion = letter_counting(string_exam)
+print(f"'{string_exam}' == '{conclusion}'")
```
Comment (Stml89) [inline homework9part3.py]:
```suggestion
assert letter_counting("cccbba") == "c3b2a"
.....
assert ("aaabbdefffff") == "a3b2def5"
```

**Review by Stml89 — APPROVED** [review]
Fix comments and we are good to go

File: homework9.py (hunk 3 above)
Comment (ValeriaKudelko) [reply homework9.py]: Done

File: homework9.py (hunk 4 above)
Comment (ValeriaKudelko) [reply homework9.py]: Done

File: homework9part3.py (hunk 5 above)
Comment (ValeriaKudelko) [reply homework9part3.py]: Done

**Review by Stml89 — APPROVED** [review]

**Pull request merged**